samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -197,22 +215,116 @@ int main(void)
 			net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
 			sys_put_le32(iso_send_count, iso_data);
 			net_buf_add_mem(buf, iso_data, sizeof(iso_data));
-			ret = bt_iso_chan_send(&bis_iso_chan[chan], buf, seq_num);
+
+			/* Read Tx Sync interface returns error if no ISO data was given previously.
+			 */
```

Comment:
Can we avoid calling `bt_iso_chan_get_tx_sync()` when no data has been sent on a channel? Then we avoid error prints in the sample